index.html

```diff
@@ -84,107 +88,99 @@
   </div>
 </section>
 
-<!-- Two Services Section -->
-<section class="bg-tan-light watch-section" style="margin-top:-24px;">
-  <div class="container soft-bottom">
-    <div class="row text-center push-ends soft-ends">
-      <div class="col-sm-12">
-        <p class="flush-bottom font-size-small text-orange text-uppercase">
-          <strong>Watch Weekly Teachings</strong>
-        </p>
-        <h3 class="section-header flush-ends text-gray-darker">
-          TWO INSPIRATIONAL MESSAGES<br>
-          EVERY WEEK
-        </h3>
-      </div>
-    </div>
-
-    <div class="row" style="margin-right: 0; margin-left: 0;">
-
-      <!-- Live Message -->
-      <div class="col-xs-6 push-bottom" style="padding-left: 8px; padding-right: 8px;">
-        <div class="bg-white weekend-container">
-
-          <!-- Watch In Room Image - Content Block-->
-          {{ watch_in_room_image.content_block | markdownify }}
-
-          <div class="soft-sides mobile-soft-half-sides push-half-ends">
-
-            <!-- Watch In Room Details - Content Block-->
-            {{ watch_in_room_details.content_block | markdownify }}
-
-            <div class="row">
-              <div class="col-xs-12 weekend-message">
-                <hr>
+<!-- Curent Series Section -->
+<section class="bg-blue watch-section" style="margin-top:-24px;"> 
+  <div style="position: relative; background-image: url('{{ series.background_image.url | imgix: site.imgix }}?{{ site.imgix_params.placeholder }}');" data-optimize-bg-img>
+    <div style="background-image: linear-gradient(to top, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.4));">
+      <div class="container soft-bottom">
+        <div class="row text-center push-top soft-ends">
+          <div class="col-sm-12">
+            <p calss="flush-bottom font-size-small text-white text-uppercase">
+              <strong>Current Series</strong>
+            </p>
+            <h2 class="section-header flush-ends text-white">{{ series.title }}</h2>
+            <p class="push-top col-sm-10 col-sm-offset-1">{{ series.description }}</p>
+            <div class="row btn push-right">
+              <crds-button block="true" display="outline" color="white" href="/latest-weekend" text="Watch the current teaching series"></crds-button>
```

Comment:
`/latest-weekend` 404s on [int](https://int.crossroads.net/latest-weekend) and [production](https://www.crossroads.net/latest-weekend) is this a page that will be setup?